src/main/lib/file-utils.js

```diff
@@ -41,4 +41,23 @@ const listFilesInDirectory = (dir, excludes = []) => {
     );
 };
 
-module.exports = { writeJson, readJson, listFilesInDirectory, exists };
+function listFilesInDirectoryRec (root, dir, excludes = []) {
```

Comment:
What `root` and `dir` args are for? When function is invoked both receive same value.

Reply:
To be honest current implementation adds unnecessary complexity.

I recommend using `https://www.npmjs.com/package/glob` module which should be easy to understand to potential future contributors.

On top of that I recommend reimplementing logic of `json2xlsx` so that:
1. there is one call that lists all JSON files in specified directory
2. output is used grouped by top level file or directory name (without file extension)
3. grouped output is iterated over to read all the files and do processing AS IS

Reply:
`root` was the root directory from where the function was initially called, this is then used to build the relative path to the file so it can then be read. `dir` is the current directory being read to get all of the files.

Reply:
Will have a look into glob